src/doc/trpl/rust-inside-other-languages.md

```diff
@@ -66,10 +66,15 @@ threads = []
     5_000_000.times do
       count += 1
     end
+
+    Thread.current[:output] = count
```

Comment:
Probably, my ruby is not very up to par. Would it be better to just return `count` from the threads and then collect the values using `t.value`?